functions/lib/sql-helper.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const R = require('ramda')
+const sqlService = require('less-tedious')
+
+const checkFormAllocationTable = '[checkFormAllocation]'
+const { TYPES } = require('tedious')
+const schema = '[mtc_admin]'
+
+/**
+ * Retrieve the checkFormAllocation data from the db
+ * @param checkCode
+ * @return {Promise<void>}
```

Comment:
@jon-shipley JSdoc return type is wrong.
You are returning an object(R.head) as far as I see.